internal/versions/versions.go

```diff
@@ -22,56 +19,9 @@ var (
 	// because the original version of the mTLS credential was not compatible with KIC.
 	MTLSCredentialVersionCutoff = semver.Version{Major: 2, Minor: 3, Patch: 2}
 
-	// FlattenedErrorCutoff is the Kong version prior to the addition of flattened errors.
-	FlattenedErrorCutoff = semver.Version{Major: 3, Minor: 1}
-)
-
-var (
-	// kongVersion holds the Kong version singleton. If never initialized (during some tests), it defaults to the
-	// lowest possible version.
-	kongVersion = KongVersion(semver.MustParse("0.0.0"))
-
-	kongVersionOnce sync.Once
+	// FlattenedErrorCutoff is the lowest Kong version with support for flattened errors.
+	FlattenedErrorCutoff = semver.Version{Major: 3, Minor: 2}
 
-	kongVersionLock sync.RWMutex
+	// TLSPassthroughCutoff is the lowest Kong version with support for TLS passthrough.
+	TLSPassthroughCutoff = semver.Version{Major: 2, Minor: 7}
```

Comment:
Couldn't find this explicitly mentioned in kong's changelog.

This section in the docs is only present for GW 3.0+ https://docs.konghq.com/gateway/3.0.x/how-kong-works/routing-traffic/#proxy-tls-passthrough-traffic

Reply:
As it's used only in tests, I think we can assume that was true.